dot/rpc/modules/api.go

```diff
@@ -90,7 +90,7 @@ type RuntimeAPI interface {
 type SystemAPI interface {
 	SystemName() string
```

Comment:
is "SystemName" now the name of the node?

Reply:
SystemName is from ctx.App.Name, and is used for RPC system_name
ChainName is from genesis.json 'name' parameter, and is used for RPC sysytem_chain
Node name shown is telemetry is from cfg.Global.Name, from '--name' flag or a random name (after #1496 is done)